src/reyna/Connection/ConnectionInfo.cs

```diff
@@ -62,6 +62,62 @@ public bool Mobile
             }
         }
 
+        public bool Wifi
+        {
+            get
+            {
+                try
+                {
+                    var interfaces = NetworkInterface.GetAllNetworkInterfaces();
+                    foreach (var ni in interfaces)
```

Comment:
Do we really want to check all networkInterfaces, and not just the active network? In Android we only check active network.